compressor/management/commands/compress.py

```diff
@@ -148,7 +148,7 @@ def compress(self, log=None, **options):
             raise OfflineGenerationError("No template paths found. None of "
                                          "the configured template loaders "
                                          "provided template paths. See "
-                                         "http://django.me/template-loaders "
+                                         "https://docs.djangoproject.com/en/1.8/topics/templates/ "
```

Comment:
Not sure... I initial thought about linking there — but with 1.8's new features I thought the more general link...